constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/MotionLayout.kt

```diff
@@ -669,7 +657,7 @@ enum class LayoutInfoFlags {
 @PublishedApi
 internal fun rememberMotionLayoutMeasurePolicy(
     optimizationLevel: Int,
-    debug: EnumSet<MotionLayoutDebugFlags>,
+    debug: EnumSet<ConstraintSetParser.MotionLayoutDebugFlags>,
```

Comment:
Don't forget that this might be a commonly used API.

As it is, it'd require people to reference a core API.

this is a case where `typealias` might help

``` typealias MotionDebugFlag = ConstraintSetParser.MotionLayoutDebugFlags ```

So a call might be 

```kotlin
MotionLayout(
   motionScene = MotionScene(...),
   progress = progress,
   debug = EnumSet.of(MotionDebugFlag.ALL)
)
```

Reply:
put back MotionLayoutDebugFlags  
 made 
     fun getForcedDrawDebug(): MotionLayoutDebugFlags
part of interface MotionScene : CoreMotionScene {

I really do not like this pattern of wrapper interface but we have it might as well use it now.

Also I want to move away from enum in core so this is a better pattern. 
The goal is to think of Core as potential C++ and we exchange structs of stings, integers and floats. 
But we need to get there step at a time